js/github.js

```diff
@@ -9,10 +10,38 @@ const doesGitHubCookieExist = () => {
   return !!githubCookieStr;
 };
 
+const SIGN_IN_GITHUB_OAUTH = () => {
+  let originURL = window?.location?.origin ?? '/';
```

Comment:
Is optional chaining necessary ??

Reply:
You have already used the null coalescing operator, so it won't throw any error if the `window.location.origin` is undefined or null